src/Service/SF1500/SF1500.php

```diff
@@ -353,11 +344,11 @@ public function getPersonMagistrat(string $funktionsId)
 
         $enhedNavn = $this->getValue($data, $enhedsNavnKeys);
 
-        $data = $this->getEnhedNavnOgOverordnetOrganisationsId($orgEnhedId, $token);
+        $data = $this->getEnhedNavnOgOverordnetOrganisationsId($orgEnhedId);
 
         while ($orgEnhedId = $data['overordnet_id']) {
```

Comment:
I suppose this is expected to run until `$data` is empty and in that case you should use `$orgEnhedId = ($data['overordnet_id'] ?? null))` the avoid “Undefined Index” errors.

Reply:
This key is always set.